templates/Linter.html.jinja2

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>{{ site_title }}: {{ linter.name }}</title>
+  <link rel="stylesheet" href="../assets/css/coala.css" type="text/css" />
+  <link rel="stylesheet" href="../assets/css/coast.css" type="text/css" />
+</head>
+<body>
+  <section class="thin-row-small">
+    <h1>{{ linter.name }}</h1>
+  </section>
+  <table>
+    <tr>
+      <th>executable</th>
+      <td>{{ linter.executable }}</td>
+    </tr>
+    <tr>
+      <th>Repology</th>
+      <td>{{ linter.repology_id }}</td>
+    </tr>
+    <tr>
+      <th>Wikidata</th>
+      <td>{{ linter.wikidata }}</td>
+    </tr>
+    <tr>
+      <th>Packages</th>
+      <td>
+          <ul>
+              {% for package in linter.packages %}
+              <li>{{ package.manager.name }} {{ package.name }}
+              {% endfor %}
```

Comment:
Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5qkx5gpz/templates/Linter.html.jinja2
+++ b/tmp/tmp5qkx5gpz/templates/Linter.html.jinja2
@@ -28,7 +28,7 @@
           <ul>
               {% for package in linter.packages %}
               <li>{{ package.manager.name }} {{ package.name }}
-              {% endfor %}
+              {% endfor %}{# for package in linter.packages #}
           </ul>
       </td>
     </tr>
```